README.md

```diff
@@ -57,12 +57,11 @@ Islandora 8 provides an Ansible [claw-playbook](https://github.com/Islandora-Dev
 
 * [Daniel Lamb](https://github.com/dannylamb/)
 * [Jared Whiklo](https://github.com/whikloj)
```

Comment:
I [signed up](https://docs.google.com/spreadsheets/d/10KUQGfDP5x4uHnvkVtz8EV_S0pjLFxrJqSYhcMf7-C0) to be a maintainer on this one too (leaving you and Danny listed also seems prudent!)